### PR TITLE
(dbt) Update firebase model reference

### PR DESCRIPTION
**Pull request by carolinemestre-passculture**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/dependencies/analytics/sql/analytics/firebase_aggregated_search_events.sql [hunk 1]
```diff
@@ -24,7 +24,7 @@ booked_from_search AS (
         , consult_timestamp
         , delta_diversification
     FROM consulted_from_search
-    JOIN `{{ bigquery_analytics_dataset }}`.firebase_events
+    JOIN `{{ bigquery_int_firebase_dataset }}`.native_event
         ON consulted_from_search.unique_session_id = firebase_events.unique_session_id
```
Comment (LucileRainteau) [inline orchestration/dags/dependencies/analytics/sql/analytics/firebase_aggregated_search_events.sql:28]: faut update l'alias de la table

File: orchestration/dags/dependencies/sendinblue/sql/analytics/sendinblue_newsletters_performance.sql [hunk 2]
```diff
@@ -19,7 +19,7 @@ user_traffic as (
         , count(distinct case when event_name = 'ConsultOffer' then offer_id else null end) as offer_consultation_number
```
Comment (LucileRainteau) [inline orchestration/dags/dependencies/sendinblue/sql/analytics/sendinblue_newsletters_performance.sql:19]: il y a une dépendance dans le fichier sendinblue_transactional_performance.sql aussi

Comment (LucileRainteau) [reply orchestration/dags/dependencies/sendinblue/sql/analytics/sendinblue_newsletters_performance.sql:19]: et dans le fichier training_data_clicks.sql

File: orchestration/dags/dependencies/analytics/sql/analytics/firebase_aggregated_search_events.sql (hunk 1 above)
Comment (carolinemestre-passculture) [reply orchestration/dags/dependencies/analytics/sql/analytics/firebase_aggregated_search_events.sql:28]: yes bien vu !

File: orchestration/dags/common/config.py [hunk 3]
```diff
@@ -43,6 +43,9 @@
 BIGQUERY_ANALYTICS_DATASET = os.environ.get(
     "BIGQUERY_ANALYTICS_DATASET", f"analytics_{ENV_SHORT_NAME}"
 )
+BIGQUERY_INT_FIREBASE_DATASET = os.environ.get(
+    "BIGQUERY_INT_FIREBASE_DATASET", f"int_firebase__{ENV_SHORT_NAME}"
```
Comment (LucileRainteau) [inline orchestration/dags/common/config.py:47]:
```suggestion
    "BIGQUERY_INT_FIREBASE_DATASET", f"int_firebase_{ENV_SHORT_NAME}"
```

**Review by LucileRainteau — APPROVED** [review]

**Pull request merged**